paper-bottom-sheet.html

```diff
@@ -45,7 +45,7 @@
 			}
 		</style>
 
-		<paper-dialog id="dialog" class="dialog" animation-config="{{_dialogAnimationConfig}}" modal="[[modal]]" with-backdrop="[[withBackdrop]]" on-iron-overlay-closed="_attemptRemoval" auto-fit-on-attach>
+		<paper-dialog id="dialog" class="dialog" animation-config="{{_dialogAnimationConfig}}" no-cancel-on-esc-key="noCancelOnEscKey" no-cancel-on-outside-click="[[noCancelOnOutsideClick]]" modal="[[modal]]" with-backdrop="[[withBackdrop]]" on-iron-overlay-closed="_attemptRemoval" auto-fit-on-attach>
```

Comment:
Could you please add the required brackets for Polymer data binding -> `no-cancel-on-esc-key="[[noCancelOnEscKey]]"` (brackets missing)